docs/deep-dives/mobile/first-run-experiments.mdx

```diff
@@ -0,0 +1,92 @@
+
+This means that each change to which first-run experiments are live must be met with a code change to add or remove experiments from the bundled file. This would be a very frustrating process to keep up with manually, so we've created an [update-experiments github action](https://github.com/mozilla-mobile/update-experiments) to automate it. Below is an example job configuration for this action.
+
+```yaml
+jobs:
+  update-nimbus-experiments:
+    name: "Update Nimbus Experiments"
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout Main Branch"
+        uses: actions/checkout@v3
+        with:
+          path: $DIRECTORY
+          ref: main
+          fetch-depth: 0
+      - name: "Update Experiments JSON"
+        id: update-experiments-json
+        uses: mozilla-mobile/update-experiments@v3
+        with:
+          repo-path: $DIRECTORY
+          output-path: path/to/initial_experiments.json
+          experimenter-url: https://experimenter.services.mozilla.com/api/v6/experiments-first-run/
+          app-name: $APP_NAME
+          branch: automation/update-nimbus-experiments
+      - name: Create Pull Request
+        id: create-pull-request
+        uses: peter-evans/create-pull-request@v4
+        if: steps.update-experiments-json.outputs.changed == 1 && steps.update-experiments-json.outputs.changed-branch == 1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          path: $DIRECTORY
+          branch: automation/update-nimbus-experiments
+          commit-message: "update initial_experiments.json based on the current first-run experiments in experimenter"
+          title: "Update initial experiments JSON for Nimbus"
+          body: "This (automated) PR updates the initial_experiments.json on the `main` branch"
+          delete-branch: true
+```
+
+The only file type currently supported is JSON, and this is usually named `initial_experiments.json`. Some examples of how this workflow and file are used can be seen in [this PR](https://github.com/mozilla-mobile/firefox-ios/pull/12872/files) and [the `initial_experiments.json` file](https://github.com/mozilla-mobile/firefox-ios/blob/main/Client/Experiments/initial_experiments.json) in the Firefox iOS repository.
+
+### Enrollment
+
+For traditional experiments, enrollment is a short period at the beginning of a live experiment's lifecycle where the experiment can enroll new users. The Nimbus SDK does not do any date calculations when determining whether an experiment is enrolling users, instead relying on the `isEnrollmentPaused` experiment field.
+
+Because first run experiments are bundled with the application, that field will remain `true` in the application bundle for the duration of the release. As a result, **even after enrollment is paused within experimenter, first run users will still be evaluated and enrolled.**
```

Comment:
```suggestion
Because first run experiments are bundled with the application, that field will remain `true` in the application bundle for the duration of the release. As a result, **even after enrollment is paused remotely via `experimenter`, First Run Experiments on the app's first run will still be evaluated and enrolled.**
```